src/js/BookReader/Mode2Up.js

```diff
@@ -690,7 +677,9 @@ export class Mode2Up {
     }
 
     if (null == index) {
-      index = this.br.twoPage.currentIndexL + 2; // $$$ assumes indices are continuous
+      const nextPage = this.book.getPage(this.br.twoPage.currentIndexR).nextCollapsedUnviewables;
```

Comment:
perhaps: `nextAvailablePage`

Reply:
Renamed to `findNext` with args :+1: (I was thinking something like that, but the caveat was that it might still yield an unavailable page; all it was actually doing was collapsing a range of unavailable pages to just one page (the first); it wasn't skipping all unavailable pages).